pkg/clusteragent/admission/controllers/webhook/controller_v1_test.go

```diff
@@ -15,24 +15,30 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
-
-	"github.com/DataDog/datadog-agent/pkg/clusteragent/admission/mutate"
-	"github.com/DataDog/datadog-agent/pkg/config"
-	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/certificate"
-
+	"github.com/stretchr/testify/require"
 	admiv1 "k8s.io/api/admissionregistration/v1"
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes/fake"
+
+	"github.com/DataDog/datadog-agent/pkg/clusteragent/admission/mutate"
+	"github.com/DataDog/datadog-agent/pkg/config"
+	"github.com/DataDog/datadog-agent/pkg/util/kubernetes/certificate"
 )
 
+const waitFor = 5 * time.Second
```

Comment:
IMO we can be more generous given that we're using `Eventually` everywhere

Reply:
It's a bit tricky to define the right timeout. With 3s the tests only failed once in months.
With a very long timeout the downside is that when developing we'll be waiting an unnecessary long time if there's a bug in the code that makes the test fail.